src/main/resources/db/migration/V2__new_table_and_column.sql

```diff
@@ -0,0 +1,8 @@
+
```

Comment:
better change the name of the file to something more descriptive, i.e. not the what, but the why:

Reply:
suggested: `V2__added_timezone.sql`